hackathon_site/dashboard/frontend/src/slices/users/userSlice.ts

```diff
@@ -44,6 +58,16 @@ export const initialState: UserInitialState = {
         isLoading: false,
         error: null,
     },
+    userAcceptance: {
+        user: null,
+        isLoading: false,
+        error: null,
```

Comment:
We need to add a new field to record the application state? Bcz review_state will appear only if there is a review record in the database. If the user hasn't submitted their application, they won't be able to have the review_state

Reply:
And that might be the reason why we are failing the first step as I mentioned in my comment

Reply:
fixed!